app/models/miq_action.rb

```diff
@@ -873,15 +873,23 @@ def action_cancel_task(action, rec, inputs)
     vim.cancelTask(task_mor)
   end
 
+  def ae_user_object(rec)
+    rec.try(:evm_owner) || User.super_admin
```

Comment:
@lfu The `evm_owner` check is limited to VMs and Services but are there other models we need to be concerned with here?  Should this be using something like the [automate_user_ids](https://github.com/ManageIQ/manageiq/blob/master/lib/miq_automation_engine/engine/miq_ae_event.rb#L154) method in miq_ae_event.rb?